pootle/apps/pootle_app/views/admin/urls.py

```diff
@@ -50,9 +50,9 @@
         name='pootle-admin-permissions'),
 
     # XHR
-    url(r'^more-stats/?$',
-        dashboard.server_stats_more,
-        name='pootle-admin-more-stats'),
+#    url(r'^more-stats/?$',
+#        dashboard.server_stats_more,
+#        name='pootle-admin-more-stats'),
```

Comment:
dont comment out - we can add it back if we need it - but no point distributing commented code